x-pack/platform/plugins/shared/fleet/public/applications/fleet/sections/settings/components/outputs_table/index.tsx

```diff
@@ -117,6 +121,16 @@ export const OutputsTable: React.FunctionComponent<OutputsTableProps> = ({
           defaultMessage: 'Status',
         }),
       },
+      ...(enableSyncIntegrationsOnRemote
```

Comment:
hiding the column if feature flag is off

<img width="1185" alt="image" src="https://github.com/user-attachments/assets/746806ca-5f9c-43b6-b655-6936262dada9" />